cmd/sales-admin/main.go

```diff
@@ -34,11 +36,8 @@ func main() {
 	// Configuration
 
 	var cfg struct {
-		CMD string `envconfig:"CMD"`
-		DB  struct {
-			DialTimeout time.Duration `default:"5s" envconfig:"DIAL_TIMEOUT"`
-			Host        string        `default:"localhost:27017/gotraining" envconfig:"HOST"`
-		}
+		CMD  string `envconfig:"CMD"`
+		DB   database.Config
```

Comment:
Not sure I like this at all. I rather see it and the tags. This is hiding information. Let's not couple these things together.